Sources/Models/RequestModel.swift

```diff
@@ -37,4 +82,106 @@ open class RequestModel: Codable {
         responseHeaders = responseHttp.allHeaderFields as? [String: String]
     }
     
+    var curlRequest: String {
+        var components = ["$ curl -v"]
+
+        guard
+            let _ = self.host
+        else {
+            return "$ curl command could not be created"
+        }
+
+        if method != "GET" {
+            components.append("-X \(method)")
+        }
+
+        components += headers.map {
+            let escapedValue = String(describing: $0.value).replacingOccurrences(of: "\"", with: "\\\"")
+
+            return "-H \"\($0.key): \(escapedValue)\""
+        }
+
+        if let httpBodyData = httpBody, let httpBody = String(data: httpBodyData, encoding: .utf8) {
+            var escapedBody = httpBody.replacingOccurrences(of: "\\\"", with: "\\\\\"")
+            escapedBody = escapedBody.replacingOccurrences(of: "\"", with: "\\\"")
+
+            components.append("-d \"\(escapedBody)\"")
+        }
+        
+        for credential in credentials {
+            components.append("-u \(credential.0):\(credential.1)")
+        }
+        
+        if let cookies = cookies {
+            components.append("-b \"\(cookies[..<cookies.index(before: cookies.endIndex)])\"")
+        }
+
+        components.append("\"\(url)\"")
+
+        return components.joined(separator: " \\\n\t")
+    }
+    
+    var postmanItem: PMItem? {
+        guard
+            let url = URL(string: self.url),
+            let scheme = self.scheme,
+            let host = self.host
+            else { return nil }
+        
+        let dateFormatterGet = DateFormatter()
+        dateFormatterGet.dateFormat = "yyyyMMdd_HHmmss"
```

Comment:
Is this date format universal for Postman requests?

Reply:
I added these lines in order to easily inspect network calls made within the same session. Collections and items belonging to the same collection should look like this: 
<img width="394" alt="image" src="https://user-images.githubusercontent.com/37136929/79046825-5c1f4f00-7c13-11ea-9b96-7a8d80bca9d6.png">

Reply:
For the same reason, I added similar lines on ShareUtils.swift (lines 79-80) for collections.